src/MediaManagerBundle/Bridge/Chameleon/BackendModule/MediaManagerBackendModule.php

```diff
@@ -48,6 +48,8 @@ class MediaManagerBackendModule extends MTPkgViewRendererAbstractModuleMapper
 {
     const PAGEDEF_NAME = 'mediaManager';
 
+    const PAGEDEF_IFRAME_NAME = 'mediaManagerIframe';
```

Comment:
"iframe" is a technical term and is also not really what we do here. Is a term referring to "pick image" better here? This would use a domain terminology and also avoid to make hard assumptions about the used technology.

Reply:
renamed the const and pagedef file to pickImage